app/views/hold_recalls/_header.html.erb

```diff
@@ -1,3 +1,3 @@
 <h1 id='dialogTitle'>
-  <%= t("headers.hold_recall.#{i18n_title_key_for_hold_recall}", default: t('headers.hold_recall.default')) %>
+  <%= t(".#{i18n_library_title_key}", default: t(".#{i18n_location_title_key}", default: t('.default'))) %>
```

Comment:
You can write these fallbacks more succinctly as:

```
t(:".#{i18n_library_title_key}", default: [:".#{i18n_location_title_key}", :'.default'])) 
```

(being careful to use symbols -- if you use strings, they'll be treated as the string literal, not the i18n key)